src/common/thumbnail-manager.js

```diff
@@ -37,7 +37,8 @@ class ThumbnailManager {
 
   _convertKalturaThumbnailToThumbnailInfo = (time: number): ?ThumbnailInfo => {
     if (this._thumbnailConfig) {
-      const {thumbsSprite, thumbsWidth, thumbsHeight, thumbsSlices} = this._thumbnailConfig;
+      const {thumbsSprite, thumbsWidth, thumbsSlices} = this._thumbnailConfig;
+      const {thumbsHeight} = DefaultThumbnailConfig;
```

Comment:
so the height is not configurable anymore ? why?

Reply:
As per our patch with the backend cache issue, we are not adding the height to the thumbnail service request, so no sense of make it configurable.

Reply:
you mean the height is fixed when we getting the thumbnail from kaltura? 
this is because kaltura always returns one line sprite and not matrix? and the thumb height is the sprite height?
so maybe better to remove height from here at all and ThumbnailInfo should return the sprite height for this 
I just wonder if kaltura BE will change the default height we get incorrect value here

Reply:
The `ThumbnailInfo` shouldn't return the sprite height since thumbnail info returns info regarding specific thumbnail. As a consumer of this API you just want to know what are the thumb's dimensions and coordinates.
Thumb height is something that included in the data of the `ImageTrack`. In this case of Kaltura thumbnail, we are still not creating image track. 
As for the BE, what you are saying was true even before this change, as we used hard-coded height and width in the UI. This is why in the original implementation I was added the call with explicit values.

Reply:
i meant when the sprite is one line the sprite height is actually is the thumb height so why not to use it?

Reply:
How do u know what is the actual height?